bookmarked.html

```diff
@@ -27,20 +29,22 @@ <h1 class="header__headline">Quiz App</h1>
           </svg>
         </button>
         <span>Will there be a next question?</span>
-        <button class="card__button" class="card__button_show">
+        <button class="card__button" class="card__button--show">
```

Comment:
Usually you set the `class` attribute only once and separate CSS classes with a space. :) And I think I did not see a CSS rule for `card__button-show`. So maybe this class is not necessary for now? 🤔